backend/entities/Project.cs

```diff
@@ -111,6 +111,23 @@ public class Project : BaseEntity
         public string AgencyResponseNote { get; set; }
         #endregion
 
+        #region SPL
+        /// <summary>
+        /// get/set - Note summerizing offers received.
+        /// </summary>
+        public string OffersNote { get; set; }
```

Comment:
Where you planning to move this to the agency response table later?

Reply:
Time permitting. Focusing on getting something matching the lofi first.